test/unit-test/dawn/SIR/TestSIRComparison.cpp

```diff
@@ -0,0 +1,129 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                          _
+//                         | |
+//                       __| | __ ___      ___ ___
+//                      / _` |/ _` \ \ /\ / / '_  |
+//                     | (_| | (_| |\ V  V /| | | |
+//                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#include "dawn/SIR/AST.h"
+#include "dawn/SIR/ASTUtil.h"
+#include "dawn/SIR/SIR.h"
+#include "dawn/Support/Casting.h"
+#include "dawn/Support/STLExtras.h"
+#include "llvm/ADT/STLExtras.h"
+#include <gtest/gtest.h>
+#include <iostream>
+
+using namespace dawn;
+
+namespace {
+
+TEST(SIRComparisonTest, Dummy) {
+  /// Works all the time, initializing the suite
```

Comment:
In general `//` should be used for comments and `///` if you want to write a Doxygen comment.